action.yml

```
@@ -30,28 +30,33 @@ runs:
         # times in different directories)
         mkdir -p /tmp/gaproot/pkg/
         ln -f -s $PWD /tmp/gaproot/pkg/
+
+        # start GAP with custom GAP root, to ensure correct package version is loaded
+        GAPROOT=${GAPROOT-$HOME/gap}
+        echo "GAP=$GAPROOT/gap -l /tmp/gaproot; --quitonbreak" >> "$GITHUB_ENV"
+        echo "GAPROOT=$GAPROOT" >> "$GITHUB_ENV"
     - name: "Check for GAP manual"
       shell: bash
       run: |
-        cd $HOME/gap/doc/ref
+        cd $GAPROOT/doc/ref
         if [ ! -f manual.six ]; then
-          cd $HOME/gap
-          make html
+          cd $GAPROOT
+          make html || true
```

I think this is the central line in this PR. I would prefer if there was a comment explain why it's there. It can be brief and then link to https://github.com/gap-actions/build-pkg-docs/issues/22 for details.

Instead of `true` one could also use the null command `:`
```suggestion
          # build a HTML version of the GAP reference manual to allow subsequent 
          # steps to pass when building a package manual that contains refs to
          # the GAP reference manual.
          # There is a caveat, though: building the GAP reference manual in turn
          # can fail if it contains undefined references; this can happen because
          # the GAP manual contains references to several packages and we don't
          # install a full package distribution when running tests.
          # See also <https://github.com/gap-actions/build-pkg-docs/issues/22>.
          make html || :
```